cmd/ledger/import.go

```diff
@@ -24,8 +28,9 @@ func NewDefaultImportStore() *ImportStore {
 
 func NewImportController() *ImportController {
 	return &ImportController{
-		store:         NewDefaultImportStore(),
-		inputFileFlag: "file",
+		store:              NewDefaultImportStore(),
+		inputFileFlag:      "file",
+		recoverFromLastLog: "recover-from-last-log",
```

Comment:
Maybe `resume-from-last-log` is a better name for the flag?

Reply:
Yes, it make sense. I'll change.